src/rp2_common/hardware_watchdog/include/hardware/watchdog.h

```diff
@@ -74,14 +74,21 @@ void watchdog_update(void);
  * \note If \ref watchdog_start_tick value does not give a 1MHz clock to the watchdog system, then the \p delay_ms
  * parameter will not be in milliseconds. See the datasheet for more details.
  *
+ * \if rp2040_specicifc
```

Comment:
spelling

Reply:
:man_facepalming:  I guess that's what happens when I make late-night PRs!